.github/workflows/build-push-deploy.yml

```diff
@@ -44,12 +44,16 @@ jobs:
         username: obolnetwork
         password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+    - name: Maybe overwrite app/version.Version with git tag
```

Comment:
why `Maybe`?

Reply:
this means we do not do this for normal main branch commits. In that case, we just use the default `app/version.Version` value in the code.